pandas/tests/computation/test_eval.py

```diff
@@ -1105,39 +1105,11 @@ def test_single_variable(self):
         df2 = self.eval("df", local_dict={"df": df})
         tm.assert_frame_equal(df, df2)
 
-    def test_truediv(self):
+    def test_truediv_removed(self):
```

Comment:
i think we're OK just removing this test

Reply:
removed